CHANGELOG.md

```diff
@@ -1 +1 @@
-See: https://www.elastic.co/guide/en/elasticsearch/client/python-api/master/release-notes.html
+See: https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/release-notes.html
```

Comment:
We'll need to change this again to point to the new docs, right?

Reply:
Yes, at least for main and 9.0. Which makes me wonder if we should point to a local file in the repo instead of an URL